src/App.tsx

```diff
@@ -20,12 +25,17 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={(event) => {
+                  setValue(event.target.value);
+                }}
               />
             </div>
           </div>
-        </div>
+          {query === '' && filterdMovies}
```

Comment:
Here you're checking the query for both 'if its empty, and 'if it's not empty. 
So I thought you can just replace these checks with the part from previous comment

It'd be:

```ts
  </div>
  <MoviesList movies={moviesFromServer} value={query} />
</div>

Reply:
Done, thanks